transports/websocket/Cargo.toml

```diff
@@ -3,15 +3,15 @@ name = "libp2p-websocket"
 edition = "2021"
 rust-version = { workspace = true }
 description = "WebSocket transport for libp2p"
-version = "0.42.0"
+version = "0.42.1"
```

Comment:
You'll need to bump this also in the root `Cargo.toml`.

Reply:
Will do!

This is not technically required by the rules of caret dependencies and I was not sure how we want the impact of this update to affect the rest of the crates that are dependent on this one. Should we issue changelog updates there and version bumps too?

Reply:
Resolving this to make required checks happy.